docs/advanced/document-store/subindex.md

```diff
@@ -0,0 +1,159 @@
+(subindex)=
+# Subindices
+
+Subindices allow you to efficiently search through multimodal or nested Documents.
```

Comment:
I think that we are missing a phrase to explain how the subindices allow you to bet efficient.
We need to state that subindices is a different da that the main one and that's why it is more efficient. Maybe a schema in needed.